components/ExportTransactionsCSVModal.tsx

```diff
@@ -336,6 +304,26 @@ const ExportTransactionsCSVModal = ({
               </Box>
             );
           })}
+
+        {parseToBoolean(getEnvVar('LEDGER_SEPARATE_TAXES_AND_PAYMENT_PROCESSOR_FEES')) && (
+          <div className="mt-4 flex flex-row items-center justify-between rounded-lg border p-4">
+            <div className="space-y-0.5">
+              <Label className="text-base">
+                <FormattedMessage defaultMessage="Separate transactions compatibility" />
+              </Label>
+              <p className="text-sm text-muted-foreground">
+                <FormattedMessage defaultMessage="Export taxes and payment processor fees as columns" />
+              </p>
+            </div>
+            <Switch
+              checked={flattenTaxesAndPaymentProcessorFees === true}
+              onCheckedChange={checked =>
+                checked ? setFlattenTaxesAndPaymentProcessorFees(true) : setFlattenTaxesAndPaymentProcessorFees(false)
+              }
```

Comment:
Possible simplification

```suggestion
              checked={flattenTaxesAndPaymentProcessorFees}
              onCheckedChange={setFlattenTaxesAndPaymentProcessorFees}
```